### ISSUE TITLE
UI and UX design begun with the customer section.

### ISSUE THREAD

**Issue by tegaistega** [TidotEngineering/Admin#1]
The development Gantt chart says we are to begin with the admin section..

Duty Engineers - Gideon & Victor

**Comment by Eferigho** [comment]
I think we should advise them to push their work progress to the git repo

**Comment by tegaistega** [comment]
I agree.... I just spoke with Blessing and Taofeek...  @htolajide  has said he would share the services and assign tasks to us so that we can start building the apis... He said we should get this by tomorrow 22nd June... @htolajide please confirm

**Comment by tegaistega** [comment]
> I think we should advise them to push their work progress to the git repo

**Comment by tegaistega** [comment]
> I think we should advise them to push their work progress to the git repo

I agree.... I just spoke with Blessing and Taofeek...  @htolajide  has said he would share the services and assign tasks to us so that we can start building the apis... He said we should get this by tomorrow 22nd June... @htolajide please confirm
